[DOCS] reorganizes content under the 1.0 Set up a GX environment topic (#9930)
great-expectations/great_expectations: docs/docusaurus/docs/core/set_up_a_gx_environment/_install_gx/_gx_cloud_installation.md

> GX Cloud provides a web interface for using GX to validate your data without creating and running complex Python code.  However, GX 1.0 is also capable of connecting to a GX Cloud account should you wish to engage in further customization or automation of your workflows through Python scripts.

Reviewer: ```suggestion
GX Cloud provides a web interface for using GX to validate your data without creating and running complex Python code. However, GX 1.0 can connect to a GX Cloud account if you want to customize or automate your workflows through Python scripts.
```
Should you be using a variable for GX 1.0?

Find and remove all instances of _wish_.

![image](https://github.com/great-expectations/great_expectations/assets/104205257/8f5c993b-e84c-4b27-b9b2-6357510e4419)

## Installation and setup

> To deploy a GX Agent, which serves as an intermediary between GX Cloud's interface and your organization's data stores, see [Connect GX Cloud](/cloud/connect/connect_lp.md).  The GX Agent serves all GX Cloud users within your organization.  If a GX Agent has already been deployed for your organization, you can use the GX Cloud online application without further installation or setup.

Reviewer: ```suggestion
To deploy a GX Agent, which serves as an intermediary between GX Cloud's interface and your organization's data stores, see [Connect GX Cloud](/cloud/connect/connect_lp.md). The GX Agent serves all GX Cloud users within your organization.  If a GX Agent has already been deployed for your organization, you can use the GX Cloud online application without further installation or setup.
```
Correct all double spacing.
